source/ed/ed2nav.cpp

```diff
@@ -261,6 +261,18 @@ bool rename_file(const std::string& source_name, const std::string& dest_name) {
     return true;
 }
 
+bool remove_file(const std::string& filename) {
+    if(!boost::filesystem::exists(filename))
```

Comment:
if (..) {
 // blabla
}

Reply:
Do you really need this ?
```cpp
// if the filename doesn't exist, it's already deleted....
```

Reply:
I mean, you need to put the braces after the if, even if you have one instruction after :stuck_out_tongue:

Reply:
oooh, I see!
 
It's weird, I thought `clang-format` would do that type of formatting automatically :thinking:
I think we should rather harden the clang-formatting config to enforce this behaviour.

Reply:
I thought it would too, or maybe it is clang-tidy... :thinking:

Reply:
yup, looks like it's a clang-tidy thing - https://clang.llvm.org/extra/clang-tidy/checks/readability-braces-around-statements.html